views/lexicon/includes/notifications.html.twig

```diff
@@ -37,7 +36,7 @@
 
                         <p>We've updated our user manual to feature our new and improved user interface. If you need a copy, drop us a message and we'll be happy to send one over.</p>
 
-                        <p><i class="icon-file-text-alt"></i> <a href="#">View detailed release notes</a></p>
+                        <p><i class="icon-file-text-alt"></i> <a href="#path-to-release-notes">View detailed release notes</a></p>
```

Comment:
aria-hidden true on icons

Reply:
Should be using helpers really, but it's only an example and we need to improve notifications all round